manifests/charts/gateways/istio-ingress/templates/injected-deployment.yaml

```diff
@@ -118,6 +122,10 @@ spec:
 {{ toYaml $gateway.additionalContainers | indent 8 }}
 {{- end }}
       volumes:
+      - emptyDir:
```

Comment:
not needed here since its in [gateway-injection-template.yaml](https://github.com/istio/istio/pull/37947/files#diff-34a675f455a18d4c77afb1cb2bc877a0a77315b3f8dfb8920ddf2f5af96cecb0)